docs/guides/Checkout/configuration-section-api-quick-start-guides/get-and-update-the-seller-exchange-window.md

```diff
@@ -2,36 +2,35 @@
+In such situations, VTEX allows the marketplace to have a period (window) after the customer makes the purchase to change sellers and avoid order cancellation. For more information on the process of changing sellers, please read the guides in the [Help Center](https://help.vtex.com/en/tutorial/change-seller--5TBAwO2kOAMw44uyaaQMQO) and the [Developer Portal](https://developers.vtex.com/vtex-rest-api/docs/change-seller).
+> ⚠️ The default window for changing a seller is <strong>2 days</strong>. However, you can change it for up to <strong>30 days</strong>.
 
->⚠️ The default window for the change seller is <strong>2 days</strong>. However, you can modify the value for up to a maximum period of <strong>30 days</strong>.
 
+This guide describes how to get a seller change window or update the number of days in a marketplace on your account.
 
-This guide describes how to get a seller's current exchange window or update this value in a marketplace on your account.
+## Getting the current seller change window
 
-## Getting the current window to change seller
-
-To get the window to change seller in a marketplace, you need to use the [Get window to change seller](https://developers.vtex.com/vtex-rest-api/reference/getwindowtochangeseller) endpoint as a **GET** request. In this request, you must send the `accountname` through the URL address, as shown by the example below:
+To get seller change window in a marketplace, you need to use the [Get window to change seller](https://developers.vtex.com/vtex-rest-api/reference/getwindowtochangeseller) endpoint as a **GET** request. In this request, you must send the `accountname` in the URL address, as in the example below:
 
 **Method: GET**
 
 `https://{accountName}.{environment.com.br}/api/checkout/pvt/configuration/window-to-change-seller`
 
-After sending the request, the endpoint will return the response body showing the maximum current period of time (days after the customer's purchase) allowed to perform the seller exchange:
+After sending the request, the endpoint will return the response body showing the maximum current period (days after the customer's purchase) allowed to perform the seller change:
 
 ```json
 "4"
 ```
 
-## Updating the window to change seller
+## Updating the window to change sellers
 
-To update the seller's exchange window (in days), you need to use a similar request of the **Get window to change seller**, but as a **POST** request ([Update window to change seller](https://developers.vtex.com/vtex-rest-api/reference/getwindowtochangeseller)), as shown by the example below:
+To update the seller change window (in days), you need to use a request similar to **Get window to change seller**, but as a **POST** request ([Update window to change seller](https://developers.vtex.com/vtex-rest-api/reference/getwindowtochangeseller)), as in the example below:
```

Comment:
Atualizar link: [Update window to change seller](https://developers.vtex.com/docs/api-reference/orders-api#post-/api/checkout/pvt/configuration/window-to-change-seller)